<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDESigCheckDialog</class>
 <widget class="QDialog" name="CMDESigCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="verifierOKButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="operatorOKButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>-1</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>51</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>帐户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="operatorAccountEdit">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>51</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="operatorPasswordEdit">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>84</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>117</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="operatorCommentEdit">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>117</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>84</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>51</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>帐户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="verifierPasswordEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>84</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>117</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>说明</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="verifierCommentEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>117</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="verifierAccountEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>51</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>84</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>operatorAccountEdit</tabstop>
  <tabstop>operatorPasswordEdit</tabstop>
  <tabstop>operatorCommentEdit</tabstop>
  <tabstop>operatorOKButton</tabstop>
  <tabstop>verifierAccountEdit</tabstop>
  <tabstop>verifierPasswordEdit</tabstop>
  <tabstop>verifierCommentEdit</tabstop>
  <tabstop>verifierOKButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
